perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java

```diff
@@ -95,17 +98,35 @@
 
 	final static Logger log = LoggerFactory.getLogger(PerunBlImpl.class);
 
+	final static List<String> dontLookupUsersForLogins = new ArrayList<String>();
```

Comment:
Rather to use HashSet instead of ArrayList. HashSet can perform contains method in O(1), but for ArrayList it's O(n)